kernel/kernel-api/src/main/java/io/delta/kernel/metrics/TransactionMetricsResult.java

```diff
@@ -65,5 +67,10 @@ public interface TransactionMetricsResult {
    */
   long getTotalRemoveFilesSizeInBytes();
 
-  // TODO add fileSizeHistogram
+  /**
+   * @return the file size histogram information for the table version committed in this
+   *     transaction. For a failed transaction this metric may be incomplete.
+   */
+  @JsonIgnore
```

Comment:
This would really clog up the serialization so excluding it

Reply:
Is it serialzied for the log messages?

Reply:
I purposely excluded it since it's like a 3x95 array. If you think it should be there we can discuss but either way I'd make that change in a follow up